cmd/openshift-installer/main.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	"os"
+
+	log "github.com/Sirupsen/logrus"
+	"gopkg.in/alecthomas/kingpin.v2"
+
+	"github.com/openshift/installer/pkg/asset"
```

Comment:
I like these new package paths, avoiding our current [`github.com/openshift/installer/installer/...`][1] redundancy.  But to get CI tests for these new locations, we'll need updates to [`go-lint.sh`][2], [`go-vet.sh`][3], [ci-operator][4] (also [here][5]), etc.  I'm not sure about the best way to roll those out.  Maybe something like:

1. Land an installer PR with a stub `cmd/openshift-installer/main.go` and `pkg/` directory with the associated local CI updates.
2. Land a release PR with updates to add tests for the new locations.
3. Come back and rebase this PR around the stubs from step 1.

[1]: https://github.com/openshift/installer/blob/1c2e0746bc94acff8c9ae3485b12dfc8dc23eae2/installer/pkg/config-generator/generator.go#L21
[2]: https://github.com/openshift/installer/blob/1c2e0746bc94acff8c9ae3485b12dfc8dc23eae2/hack/go-lint.sh#L2
[3]: https://github.com/openshift/installer/blob/1c2e0746bc94acff8c9ae3485b12dfc8dc23eae2/hack/go-vet.sh#L3-L11
[4]: https://github.com/openshift/release/blob/4f2e250912dc83331e11dccec3b5544e38204fca/ci-operator/jobs/openshift/installer/openshift-installer-presubmits.yaml#L221
[5]: https://github.com/openshift/release/blob/4f2e250912dc83331e11dccec3b5544e38204fca/ci-operator/config/openshift/installer/master.json#L44

Reply:
> Will it be OK to add the CI updates after this PR lands?

I've filed openshift/release#1289 bumping unit and go-lint to cover the new directories.  And I've filed #185 with the local `hack/` changes.